addons/account/models/account_account.py

```diff
@@ -372,6 +373,18 @@ def _inverse_code(self):
         }
         self.env['ir.property'].with_company(self.env.company.root_id).sudo()._set_multi('code', 'account.account', values)
 
+    @api.depends_context('company')
+    @api.depends('code')
+    def _compute_placeholder_code(self):
+        for record in self:
+            if record.code:
+                record.placeholder_code = record.code
+            elif authorized_company_ids := (set(record.company_ids.ids) & set(self.env.user._get_company_ids())):
+                placeholder_code_company = next(iter(authorized_company_ids))
+                record.placeholder_code = f'{record.with_company(placeholder_code_company).code} ({placeholder_code_company.name})'
```

Comment:
Isn't `placeholder_code_company` and `int`?
How does `placeholder_code_company.name` work? :zany_face:

Reply:
Well, *insert long unconvincing explanation here* ...

... it doesn't :laughing: